Nightmare/Locales/ptBR.lua

```diff
@@ -1,30 +1,30 @@
 local L = BigWigs:NewBossLocale("Cenarius", "ptBR")
 if not L then return end
 if L then
-	--L.forces = "Forces"
-	--L.bramblesSay = "Brambles near %s"
+	L.forces = "Forças do pesadelo"
+	L.bramblesSay = "Espinheiras perto de %s"
 end
 
 L = BigWigs:NewBossLocale("Elerethe Renferal", "ptBR")
 if L then
-	--L.isLinkedWith = "%s is linked with %s"
-	--L.yourLink = "You are linked with %s"
-	--L.yourLinkShort = "Linked with %s"
+	L.isLinkedWith = "%s esta linkado com %s"
+	L.yourLink = "Você esta linkado com %s"
+	L.yourLinkShort = "Linkado com %s"
 end
 
 L = BigWigs:NewBossLocale("Il'gynoth", "ptBR")
 if L then
-	--L.custom_off_deathglare_marker = "Deathglare Tentacle marker"
-	--L.custom_off_deathglare_marker_desc = "Mark Deathglare Tentacles with {rt6}{rt5}{rt4}{rt3}, requires promoted or leader.\n|cFFFF0000Only 1 person in the raid should have this enabled to prevent marking conflicts.|r\n|cFFADFF2FTIP: If the raid has chosen you to turn this on, having nameplates enabled or quickly mousing over the spears is the fastest way to mark them.|r"
+	L.custom_off_deathglare_marker = "Marcador de Tentáculo Fulgor da Morte"
+	--read all this line --L.custom_off_deathglare_marker_desc = "Marca Tentáculo FUlgor da Morte com {rt6}{rt5}{rt4}{rt3}, requer assistente ou líder.\n|cFFFF0000Apenas 1 pessoa no raide deve ter isto ativado para prevenir conflitos na marcação.|r\n|cFFADFF2FDICA: Se o raide escolheu você para isso, ter placas de identificação ativadas ou passar rapidamente o mouse sobre *******"the spears"(you mean the body of the tentacle?, if yes then you can use "as cabeças")**** é a maneira mais rápida de marcá-las.|r"

```

Comment:
It should say tentacles, the English version is wrong. You also have a small typo: "FUlgor"